sql/hive/src/main/scala/org/apache/spark/sql/hive/test/TestHive.scala

```diff
@@ -151,7 +155,7 @@ private[hive] class TestHiveSparkSession(
     new TestHiveSessionState(self)
 
   override def newSession(): TestHiveSparkSession = {
```

Comment:
You can change it to override `def newSession(inheritSessionState: Boolean)` instead